### DISCUSSION TITLE
rustdesk prevents hibernation

### DISCUSSION THREAD

**Discussion by Roofoo-K** [rustdesk/rustdesk#8488] [General]
### Bug Description

since version 1.2.6, rustdesk prevents sleep mode by timer and manually. It works through other remote access programs. 

### How to Reproduce

easy

### Expected Behavior

can add this function in the form of threshold settings if someone needs this behavior

### Operating system(s) on local side and remote side

windows 10

### RustDesk Version(s) on local side and remote side

1.2.6

### Screenshots

![photo_2024-06-27_02-20-44](https://github.com/rustdesk/rustdesk/assets/141381674/a3ed2953-3f9c-4bab-b40b-288ed343f39d)

**Comment by rustdesk** [comment]
close to favor https://github.com/rustdesk/rustdesk/discussions/8473#discussioncomment-9891505
